signale.js

```diff
@@ -257,6 +257,19 @@ class Signale {
     this._log(this._buildSignale(this._types[type], ...messageObj), this._types[type].stream);
   }
 
+  _padSpace(str, targetLength) {
+    if (String.prototype.padEnd) {
+      return str.padEnd(targetLength);
+    }
+    targetLength >>= 0; // Floor if number or convert non-number to 0;
+    if (str.length > targetLength) {
```

Comment:
should also return when equal;

```js
if (str.length >= targetLength) {
  return String(str);
}
```

in addition, this check should occur prior to the `if (String.prototype.padEnd)` one, since there is no need to call the method when `targetLength` is less than or equal to the length of `str`